curriculum/challenges/english/25-front-end-development/review-bash-commands/6724e387ee098d1ef33108ba.md

```diff
@@ -23,42 +23,43 @@ dashedName: review-bash-commands
 
 - **Up/Down arrows**: Cycle through previous/next commands in history.
 - **Tab**: Autocomplete commands.
-- `Control+L` (Linux/macOS) or typing `cls` (Windows): Clear the terminal screen.
-- `Control+C`: Interrupt a running command (also used for copying in PowerShell if text is selected).
-- `Control+Z` (Linux/macOS only): Suspend a task to the background; use `fg` to resume it.
-- `!!`: Instantly rerun the last executed command.
+- **`Control+L`** (Linux/macOS) or typing `cls` (Windows): Clear the terminal screen.
+- **`Control+C`**: Interrupt a running command (also used for copying in PowerShell if text is selected).
+- **`Control+Z`** (Linux/macOS only): Suspend a task to the background; use `fg` to resume it.
+- **`!!`**: Instantly rerun the last executed command.
 
 ## Bash Basics
 
 - **Bash** (Bourne Again Shell): Widely used Unix-like shell.  
   Key commands:
 
-  - `pwd`: Show the current directory.
-  - `cd`: Change directories.
-    - `..` refers to the parent directory (one level up).
-    - `.` refers to the current directory.
-  - `ls`: List files and folders.
-    - `-a`: Show all files, including hidden files.
-    - `-l`: Show detailed information about files.
-  - `less`, `more`, and `cat`: View file contents. (`more` shows one screen at a time.)
-  - `mkdir`: Create a new directory.
-  - `rmdir`: Remove an empty directory.
-  - `touch`: Create a new file.
-  - `mv`: Move or rename files.
+  - **`pwd`**: Show the current directory.
+  - **`cd`**: Change directories.
+    - **`..`** refers to the parent directory (one level up).
+    - **`.`** refers to the current directory.
+  - **`ls`**: List files and folders.
+    - **`-a`**: Show all files, including hidden files.
+    - **`-l`**: Show detailed information about files.
+  - **`less`**: View file contents one page at a time with navigation options, including scrolling backward and searching.
+  - **`more`**: Display file contents one screen at a time, but only allows scrolling forward.
```

Comment:
`more` does let you move backward using the `b` key. However, the navigation capabilities are limited as compared to less. Here is my suggestion:

```suggestion
  - **`more`**: Display file contents one screen at a time, with limited backward scrolling and basic navigation.
```